console/console-init/ui/src/components/UpdatePassword/UpdatePassword.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import { useStoreContext, types } from "context-state-reducer";
 import { PasswordInputFieldWithToggle } from "components";
 
-export const UpdatePassword: React.FC<{}> = () => {
+export const UpdatePassword = () => {
```

Comment:
Any particular reason of not having this variable as a react component?

Reply:
Reason is we don't have interface here, so we don't need React.FC as such. There will be no logical effect for adding or not adding.